java/src/org/openqa/selenium/remote/service/DriverService.java

```diff
@@ -437,40 +437,32 @@ protected Duration getDefaultTimeout() {
       return DEFAULT_TIMEOUT;
     }
 
-    protected OutputStream getLogOutput(String logProperty) {
-      if (logOutputStream != null) {
-        return logOutputStream;
-      }
-
+    protected OutputStream getLogOutput() {
       try {
-        File logFileLocation = getLogFile();
-        String logLocation;
-
-        if (logFileLocation == null) {
-          logLocation = System.getProperty(logProperty);
-        } else {
-          logLocation = logFileLocation.getAbsolutePath();
-        }
-
-        if (logLocation == null) {
-          return ByteStreams.nullOutputStream();
-        }
-
-        switch (logLocation) {
-          case LOG_STDOUT:
-            return System.out;
-          case LOG_STDERR:
-            return System.err;
-          case LOG_NULL:
-            return ByteStreams.nullOutputStream();
-          default:
-            return new FileOutputStream(logLocation);
-        }
+        return logFile != null ? new FileOutputStream(logFile) : logOutputStream;
```

Comment:
nit: Use `getLogFile` to get the value.

Reply:
I guess I "fixed" this because I need to toggle off output stream not file. 😂